cloud/google/machineactuator.go

```diff
@@ -285,9 +290,17 @@ func (gce *GCEClient) Create(cluster *clusterv1.Cluster, machine *clusterv1.Mach
 				Items: metadataItems,
 			},
 			Tags: &compute.Tags{
-				Items: []string{"https-server"},
+				Items: tags,
 			},
 			Labels: labels,
+			ServiceAccounts: []*compute.ServiceAccount{
```

Comment:
Need some context, why do we need a specified service account now?

Reply:
There was some error in the API server when spinning up the cluster that said it couldn't fetch the initial token and that it couldn't find the default service account. I don't remember the error message exactly, but it was preventing the API server from starting. I think @mkjelland was working on replacing this default service account with something more specific, though.

Reply:
I don't think every node needs this account. Just the masters since they are the ones that will be making GCE API calls.

Reply:
I think we need the service account because the controller manager will be using the GCE api to handle load balancers and pod CIDR routing.